extensions/wikia/PhalanxII/fallback/hooks/TitleBlock.class.php

```diff
@@ -90,28 +90,28 @@ static public function genericTitleCheck( $titleObj, &$block ) {
 	static public function checkTitle( Title $title, &$block ) {
 		wfProfileIn( __METHOD__ );
 
-		if (is_null(self::$blocksData)) {
+		if ( is_null( self::$blocksData ) ) {
 			self::$blocksData = PhalanxFallback::getFromFilter( PhalanxFallback::TYPE_TITLE );
 		}
 		$fullText = $title->getFullText();
-		$result = array('blocked' => false);
+		$result = array( 'blocked' => false );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "blocked" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)